### DISCUSSION TITLE
Database Help - Update Rows Automatically

### DISCUSSION THREAD

**Discussion by Abareyellow** [community/community#119731] [Programming Help]
### Body

Hello Everyone,

I have a few questions about creating a database. So I am working on a project to calculate vacation time/ sick leave for employees. After the couple of months they get a certain amount of hours they can use. The problem I am running into is creating the database table for tracking hours. I want the database to automatically update periodically to help calculate hours. I have look up something and it suggest using a SQL Trigger or SQL Procedure. Is their a better way of doing this or this the way to go? If you can, please provide code or instructions to help build this table?

Thanks

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Salvatore112** [comment]
Since you're using SQL, I'd higly reccomend that you take a look at Microsoft SQL Server. They have so-called SQL Server Agent jobs that look like something you're looking for:

Here's a documanation reference: http://technet.microsoft.com/en-us/library/ms181153(v=sql.105).aspx

They allow you to run some sql code on a schedule of your choosing.
